.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "5.10"
+  - "6.11.0"
```

Comment:
Based on [the docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions), could be better to simply use: `- "6"` (latest 6.x release)